src/components/QuizModal.tsx

```diff
@@ -18,56 +18,54 @@ const QuizModal: React.FC<QuizQuestion> = QuizQuestion => {
   const sadFaceUnicode = "\uD83D\uDE14";
   const answerIcon = QuizQuestion.correct ? lightBulbUnicode : sadFaceUnicode;
   return (
-    <>
-      <Modal className="modal-styles" show={QuizQuestion.show}>
-        <Modal.Header>
-          <Modal.Title>
-            <h2 className="modal-text">
-              {answerIcon} {QuizQuestion.message}
-            </h2>
-            <h3 className="modal-text" data-testid="modal-points">
-              Points: {QuizQuestion.points}
-            </h3>
-          </Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <p>
-            <b>Your Answer:</b>
-          </p>
-          <p>{QuizQuestion.chosenAnswer}</p>
-          <p>
-            <b>Answer:</b>
-          </p>
-          <p>{QuizQuestion.displayExplanation}</p>
+    <Modal show={QuizQuestion.show}>
```

Comment:
For the actual markup here, we don't need the Modal.Header, Modal.Title, etc stuff. 
That was something specific to React bootstrap
https://react-bootstrap.netlify.app/docs/components/modal/

But also, we don't need to recreate it either.
We can use the dialog element like this and use vanilla CSS for the styles in a `QuizModal.css` file

```
import React, { MouseEventHandler } from "react";
import "../stylesheets/QuizModal.css";

interface QuizQuestion {
  chosenAnswer: string;
  message: string;
  points: number;
  correct: boolean;
  displayExplanation: string;
  showReference: string;
  nextQuestion: MouseEventHandler;
  show: boolean;
}

const QuizModal: React.FC<QuizQuestion> = QuizQuestion => {
  const lightBulbUnicode = "\uD83D\uDCA1";
  const sadFaceUnicode = "\uD83D\uDE14";
  const answerIcon = QuizQuestion.correct ? lightBulbUnicode : sadFaceUnicode;
  return (
    <dialog open={QuizQuestion.show}>
      {/*TODO: Add appropriate classes and CSS to make the modal look like the one in production. */}
      <h2 className="modal-text">
        {answerIcon} {QuizQuestion.message}
      </h2>
      <h3 className="modal-text" data-testid="modal-points">
        Points: {QuizQuestion.points}
      </h3>

      <p>
        <b>Your Answer:</b>
      </p>
      <p>{QuizQuestion.chosenAnswer}</p>
      <p>
        <b>Answer:</b>
      </p>
      <p>{QuizQuestion.displayExplanation}</p>
      <a
        className="modal-link"
        href={QuizQuestion.showReference}
        target="_blank"
        rel="noopener noreferrer"
      >
        Learn more with this helpful resource
      </a>
      <br />
      <br />
      <p>
        Wanna learn how to code? <br /> Download the free{" "}
        <a
          className="modal-link"
          target="_blank"
          rel="noopener noreferrer"
          href="https://www.freecodecamp.org/news/learn-to-code-rpg/"
        >
          Learn to Code RPG game
        </a>
      </p>

      <button className="modal-btn" onClick={QuizQuestion.nextQuestion}>
        Next Question
      </button>
    </dialog>
  );
};
export default QuizModal;
```
We can also move the existing modal styles from the `App.css` file into the new `QuizModal.css` file